lib/chef/resource/windows_certificate.rb

```diff
@@ -496,20 +488,15 @@ def import_certificates(cert_objs, is_pfx, store_name: new_resource.store_name,
                   converge_by("Creating a PFX #{@local_pfx_path} for Store #{new_resource.store_name}") do
                     add_pfx_cert(@local_pfx_path)
                   end
-                else
-                  message = "You passed an invalid file or url to import. Please check the spelling and try again."
-                  message << exception.message
-                  raise Chef::Exceptions::ArgumentError, message
```

Comment:
Why shouldn't we raise here? The resource can't converge in this condition, right?

Reply:
oopsy, that was a mistake. Added that back in